modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -3,7 +3,7 @@
 // migration/migrating_3_4/migrating-applications-with-cam.adoc
 // migration/migrating_4_1_4/migrating-applications-with-cam.adoc
 // migration/migrating_4_2_4/migrating-applications-with-cam.adoc
-[id='migration-adding-replication-repository-to-cam_{context}']
+[id='adding-replication-repository-to-cam_{context}']
```

Comment:
Typically we do like the anchor to match the file name, so I'd recommend leaving this as it was. Same for other instances like this in the PR.

Reply:
OK. I started removing them because the links had "_migration_migration-3-4", etc., which looked odd to me. I'll change them back.